db/migrate/20151013135415_add_pub_key_to_smart_proxy.rb

```diff
@@ -0,0 +1,5 @@
+class AddPubKeyToSmartProxy < ActiveRecord::Migration
+  def change
+    add_column :smart_proxies, :pubkey, :text
```

Comment:
We do want to get this in Iteration 3 for 1.10.  I think the Foreman KeyPair would at least need to be made polymorphic. I think it could be revisited later, especially since we're probably not done with the key management topic.